### PR TITLE
Shopify 712 error renewing access token lashbox

### PR DESCRIPTION
**Pull request by andre-kraus**
# Pull Request Template

## Description

Our refresh token cron will also try to refresh tokens that can not be refreshed as even the refresh token expire. Therefor those token that error during the refresh token process can be deleted.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: extension/models/Errors/TokenRenewError.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const ETOKENRENEW = 'ETOKENRENEW'
+
+class TokenRenewError extends Error {
+  constructor (displayMessage) {
+    super('Error renewing customer access token.')
+    this.code = ETOKENRENEW
+
+    this.displayMessage = null
```
Comment (augsteyer) [inline extension/models/Errors/TokenRenewError.js]: no longer a thing anymore as far as I know, was planned to be implemented in PWA, but never was or will be

File: extension/audit-resolv.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "812|newman>postman-collection>marked": {
+    "ignore": 1
+  },
+  "812|newman>postman-runtime>postman-collection>marked": {
+    "ignore": 1
+  },
+  "755|nyc>istanbul-reports>handlebars": {
+    "remind": 1555487460723
+  }
+}
```
Comment (augsteyer) [inline extension/audit-resolv.json:11]: newline

**Review by augsteyer — CHANGES_REQUESTED** [review]

**Comment by augsteyer** [comment]
@andre-kraus a test is also needed as the coveralls are just getting smaller :P

File: extension/audit-resolv.json (hunk 2 above)
Comment (andre-kraus) [reply extension/audit-resolv.json:11]: it's auto generated by the npm-audit-resolve plugin and I don't want to mess around with it. ^^

**Review by augsteyer — APPROVED** [review]

**Pull request merged**